src/service/vaults.ts

```diff
@@ -249,11 +266,11 @@ export const watchVaultFactory = (netconfigUrl: string) => {
     }
   };
 
-  const watchManager = async (id: string) => {
-    watchGovernedParams(id);
-    watchMetrics(id);
+  const watchManager = async (path: string) => {
```

Comment:
better for this to take the manager id as it had been, but I see the watchers for the two subpaths wouldn't know which scheme to read from. So I think on balance this is good. But then for robustness and clarity assert the `id` matches `MANAGER_RE`.

Reply:
Added an assert in the util fn